contracts/exchange/contracts/src/MixinMatchOrders.sol

```diff
@@ -262,6 +285,121 @@ contract MixinMatchOrders is
         }
     }
 
+    /// @dev Determines whether or not an order has been fully filled and returns the result.
+    /// @param order The order that should be checked.
+    /// @return A boolean reflecting whether or not the order was fully filled
+    function _isFilled(LibOrder.Order memory order)
+        internal
+        view
+        returns (bool)
+    {
+        LibOrder.OrderInfo memory orderInfo = getOrderInfo(order);
```

Comment:
Could we move this logic into `batchMatchOrders()` so we only call `getOrderInfo()` once per order? `getOrderInfo()` computes the hash and does a couple SLOADs each time you call it, so it would be nice to minimize it.

You just need to update `OrderInfo.orderTakerAssetFilledAmount` with each match operation and use that to check if its been fully filled.